pkg/gcp/client/compute.go

```diff
@@ -72,6 +74,8 @@ type ComputeClient interface {
 	ListRoutes(ctx context.Context, opts RouteListOpts) ([]*compute.Route, error)
 	// DeleteRoute deletes the specified route.
 	DeleteRoute(ctx context.Context, id string) error
+	// InsertAliasIPRoute creates an alias IP route on the instance
+	InsertAliasIPRoute(ctx context.Context, route apisgcp.Route, defaultSecondarySubnetName string) error
```

Comment:
I don't really like the use of the apisgcp on the client level. You can use another "input type struct" for the client if you wish to. 
Either way when you call this function (in the single usage I saw), you construct a struct anyway.